src/library.js

```diff
@@ -618,6 +618,16 @@ LibraryManager.library = {
     }
 #endif // WASM_MEM_MAX
 
+#if USE_ASAN
+    newSize = Math.min(newSize, {{{ 8 * ASAN_SHADOW_SIZE }}});
```

Comment:
could use a comment here. i think i understand what it does, but future readers might not have all the context.

Reply:
Added the comment.